R/execution.R

```diff
@@ -584,15 +744,17 @@ if (conceptCheckThresholdLoc == "default"){
   checkResults <- .evaluateThresholds(checkResults = checkResults, 
                                       tableChecks = tableChecks, 
                                       fieldChecks = fieldChecks,
-                                      conceptChecks = conceptChecks)
+                                      conceptChecks = conceptChecks,
+                                      dbms = connectionDetails$dbms)
   
   countTotal <- nrow(checkResults)
   countThresholdFailed <- nrow(checkResults[checkResults$FAILED == 1 & 
                                               is.na(checkResults$ERROR),])
-  countErrorFailed <- nrow(checkResults[!is.na(checkResults$ERROR),])
+  countErrorFailed <- nrow(checkResults[checkResults$FAILED == 1 & 
+                                          !is.na(checkResults$ERROR),])
```

Comment:
Check on na `ERROR` should not be needed, as all statuses are mutually exclusive.

Reply:
The ERROR field contains an sql error message, it is not one of the four statuses.
The status field is called IS_ERROR.
countErrorFailed represents how many checks failed because of the sql error.